Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM node:16.13.1-alpine3.15@sha256:a2c7f8ebdec79619fba306cec38150db44a45b48380d09603d3602139c5a5f92 AS builder
+
+WORKDIR /app
+
+RUN [ "yarn", "set", "version", "1.22.17" ]
+
+COPY .yarn ./.yarn
+COPY /src ./src
+COPY package.json yarn.lock .yarnrc.yml ./
```

Comment:
The `.yarnrc.yml` should be copied at the same time as the other `.yarn` dotfiles.

Reply:
not sure I understand sorry @merlinc 
Do you mean copying the .yarn directory on the same line as .yarnrc.yml?

Reply:
The `.yarnrc.yml` also specifies the version of yarn being used in it.

https://github.com/alphagov/di-ipv-cri-address-front/blob/main/.yarnrc.yml

So if you didn't want to explicitly set the yarn version, this file would automatically do it for you.

Given it's connected to yarn & yarn versioning, I think its more appropriate to bundle it into the same copy block. ... But you can't because `docker COPY` doesn't work that way.

I'd suggest:

```
COPY .yarn  ./.yarn # yarn folder and any cjs
COPY .yarnrc.yml # make sure yarn is configured as expected
RUN [ "yarn", "set", "version", "1.22.17" ] # install yarn - this would error if the versions were different, as .yarnrc would disagree
```

Reply:
did you still want to use the local release file `.yarn/releases/yarn-1.22.17.cjs` where we set the version instead of just specifying `1.22.17`?